src/test/fuzz/fuzz.cpp

```diff
@@ -78,6 +78,30 @@ void FuzzFrameworkRegisterTarget(std::string_view name, TypeTestOneInput target,
 static std::string_view g_fuzz_target;
 static const TypeTestOneInput* g_test_one_input{nullptr};
 
+inline void test_one_input(FuzzBufferType buffer)
+{
+    used_g_prng = false;
+    seeded_g_prng_zero = false;
+    (*Assert(g_test_one_input))(buffer);
+    if (used_g_prng && !seeded_g_prng_zero) {
+        std::cerr << "\n\n"
+                     "The fuzz target ("
+                  << g_fuzz_target
+                  << ") used the global random state.\n\n"
+
+                     "This is acceptable, but requires the fuzz target to call \n"
+                     "SeedRandomStateForTest(SeedRand::ZEROS) at the beginning \n"
+                     "of processing the fuzz input.\n\n"
+
+                     "An alternative solution would be to avoid any use of globals.\n\n"
+
+                     "Without a solution, fuzz stability and determinism can lead \n"
+                     "to non-reproducible bugs or inefficient fuzzing.\n\n"
+                  << std::endl;
+        std::exit(EXIT_FAILURE);
```

Comment:
this needs to be a `std::abort()` (or similar), otherwise out of process fuzzers like afl will not catch this as a crash (by default at least)

Reply:
You may be able to confirm this with `AFL_CRASH_EXITCODE='1'` (i.e. tell afl++ to treat exit(1) as a crash)